cluster-provision/centos8/scripts/vm.sh

```diff
@@ -107,10 +107,14 @@ if [ -n "${BLOCK_DEV}" ]; then
   block_dev_arg="-drive format=qcow2,file=${BLOCK_DEV},if=virtio,cache=unsafe"
 fi
 
+MONITOR_SOCKET=/var/lib/mon${n}.sock
+
 exec qemu-system-x86_64 -enable-kvm -drive format=qcow2,file=${next},if=virtio,cache=unsafe ${block_dev_arg} \
   -device virtio-net-pci,netdev=network0,mac=52:55:00:d1:55:${n} \
   -netdev tap,id=network0,ifname=tap${n},script=no,downscript=no \
   -device virtio-rng-pci \
   -vnc :${n} -cpu host -m ${MEMORY} -smp ${CPU} ${QEMU_ARGS} \
   -serial pty -M q35,accel=kvm,kernel_irqchip=split \
-  -device intel-iommu,intremap=on,caching-mode=on -soundhw hda
+  -device intel-iommu,intremap=on,caching-mode=on -soundhw hda \
+  -monitor unix:${MONITOR_SOCKET},server,nowait \
```

Comment:
I should probably expose the monitor over `tcp::4444` and then forward the port from the container to the host using `gocli`, as done with vnc and ssh.